src/Simulation/Algorithms/COE/COE2RV.hpp

```
@@ -0,0 +1,109 @@
+            0.0
+        );
+
+        glm::dvec3 v_pqw(
+            -root * sin(nu),
+            root * (elements.e + cos(nu)),
+            0.0
+        );
+
+
+        // PQW -> IJK transformation matrix
+        double &cO = argp;
```

Clang (14.0.6)

[warning] variable name 'cO' is too short, expected at least 3 characters

(readability-identifier-length)